web/olga/analytics/models.py

```diff
@@ -268,8 +329,8 @@ def get_students_countries_amount(cls):
 
         Actually `Country is not specified` field is not a country, so it does not fill up in countries amount.
         """
-        _, tabular_format_countries_list = cls.get_students_per_country()
-        countries_amount = len(tabular_format_countries_list) - 1
+        for month in months:
+            countries_amount = len(month['tabular_countries_list']) - 1
```

Comment:
What about the case when `Country is not specified` was not received?

Reply:
That's right. That bug was in the previous version. I can fix it in the current branch.

Reply:
Found a bug here that the countries_amount doesn't iterates in the loop.